cartographer/io/image.cc

```diff
@@ -9,6 +9,21 @@ namespace cartographer {
 namespace io {
 namespace {
 
+uint32_t ColorToCairo(const Color& color) {
+  uint32_t r = ColorComponentToUint8(color[0]);
+  uint32_t g = ColorComponentToUint8(color[1]);
+  uint32_t b = ColorComponentToUint8(color[2]);
```

Comment:
Would it make sense to have separate types for `UInt8Color` and `FloatColor` and provide functions converting between the two?